playground/public/katas.html

```diff
@@ -4,9 +4,18 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Katas MathJax review</title>
+    <link
```

Comment:
I think we should have a link to this page from the main playground.

Reply:
Maybe later. It's for internal use. We don't want it getting a general audience.